packages/perseus/src/widgets/interactive-graphs/graphs/utils.ts

```diff
@@ -1,5 +1,35 @@
+import {vec} from "mafs";
+
 import type {CollinearTuple} from "../../../perseus-types";
-import type {Interval, vec} from "mafs";
+import type {Interval} from "mafs";
+
+export function calculateAngleInDegrees([x, y]: vec.Vector2) {
```

Comment:
There's another function that does this, `angleDegrees` in `vector.tsx`. Maybe the definitions could be deduplicated and moved to a common location?

It's too bad that the existing math functions (in `@khanacademy/kmath`) are such a poor fit for our use case that we can't just call them. I think the main problem is that they generalize to 3D (and maybe higher dimensions?) when AFAIK nothing actually needs 3D.